### PR TITLE
CRSF LQ OSD Style options

### PR DESCRIPTION
**Pull request by Kaiowarez**
adding an option for for diffrent LQ formats
`osd_lq_format = TBS` (default) shows `LQ 300` style
`osd_lq_format = MODE` shows mode:LQ (e.g. `2:100`)
`osd_lq_format = FREQ` shows freq:LQ (e.g. `150HZ:100`)

### CODE REVIEW

**Review by nerdCopter — COMMENTED** [review]
as per https://github.com/emuflight/EmuFlight/pull/377#issuecomment-703344381

File: src/main/io/osd.c [hunk 1]
```diff
@@ -488,32 +492,62 @@ static bool osdDrawSingleElement(uint8_t item)
...
-                              osdLQfinal = osdLQ;
-                              break;
-                          case 1:
-                              osdLQfinal = osdLQ + 100;
+
+                  case TBS:
+
+                    switch (osdRfMode)
+                    {
+                            case 2:
```
[the remark this answers is not in the document]
Comment (Kaiowarez) [reply src/main/io/osd.c:509]: its free for adjustment

**Comment by Quick-Flash** [comment]
is this ready? if so lets send it and drop 0.3.2

**Comment by nerdCopter** [comment]
waiting on a couple test, then will build to bintray for broader test.

**Review by nerdCopter — APPROVED** [review]
approval for v1

**Comment by gretel** [comment]
cool thanks

**Pull request merged**